src/components/number/Number.js

```diff
@@ -2,23 +2,28 @@ import maskInput from 'vanilla-text-mask';
 import _ from 'lodash';
 import {createNumberMask} from 'text-mask-addons';
 import {BaseComponent} from '../base/Base';
+import currencyData from '../../currencyData';
 
 export class NumberComponent extends BaseComponent {
   constructor(component, options, data) {
     super(component, options, data);
+    this.currencies = currencyData;
```

Comment:
Overrides should not be included in this pull request. They should be passed in with options to the form. So options.languageOverrides or something like that.

Reply:
I'm not disagree. But do you mean options creating the form it self? because is what I did before. I passed in the language list that you wanted to change, then compare the language with the current browser language and change the delimiter and decimal separator. Then I had a conversation with travis and I did that currency list, probably I did not understood travis.

Reply:
Should be something like this: https://github.com/formio/ngFormio/pull/569/files#diff-34a7bb8e8a80000cf8069d3add22ef5cR61